### PR TITLE
Django 1.7 breaking changes

### PR DESCRIPTION
**Pull request by voidlily**

### CODE REVIEW

**Comment by voidlily** [comment]
Merge this branch when 1.7 is released

File: satchless/payment/__init__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class PaymentsGatewayReceiptFormError(PaymentFailure):
     def __init__(self, form):
         error_message = u', '.join(
             [u':'.join([key, u', '.join(errors)])
-             for key, errors in form._errors.iteritems()])
+             for key, errors in form.errors.iteritems()])
```
Comment (jgaren) [inline satchless/payment/__init__.py]: There's a subtle difference in behavior here: `errors` returns everything from `_errors` except the `__all__` key; to maintain the old behavior, you'd need to merge `errors` and `non_form_errors`, placing `non_form_errors` under the `__all__` key.

**Comment by jgaren** [comment]
It looks like the payment app has south migrations in it, will have to update those for django 1.7

**Comment by jgaren** [comment]
Need to update setup.py to require django 1.7

File: satchless/payment/__init__.py (hunk 1 above)
Comment (voidlily) [inline satchless/payment/__init__.py]: This is fixed

**Comment by voidlily** [comment]
Migrations and setup.py have been fixed

**Comment by jgaren** [comment]
&#10004;

**Comment by jgaren** [comment]
Note: Do not delete this branch prior to updating styleseat/styleseat dependencies.

**Pull request merged**